varia/sass/abstracts/_responsive-logic.scss

```
@@ -162,26 +167,31 @@ $content-width-xxl: calc( #{map-deep-get($config-global, "breakpoint", "xxl")} -
 	@include media(mobile) {
 		/*rtl:ignore*/
 		margin-left: calc( 0.5 * (100vw - #{$content-width-sm}) );
+		max-width: calc( 0.5 * #{$content-width-sm} );;
```

Similar to above, let’s add a margin to the _right_ to prevent these blocks from getting too close to sibling blocks and, also remove that double `;` :-)

```suggestion
		max-width: calc( 0.5 * #{$content-width-sm} );
		/*rtl:ignore*/
		margin-right: #{map-deep-get($config-global, "spacing", "horizontal")};
```